File: data/conala-train_code-davinci-002_1666524963/Java/381.java
List<Integer> list1 = Arrays.asList(1, 2, 3);
List<Integer> list2 = Arrays.asList(4, 5, 6);
List<Integer> list3 = new ArrayList<>();

for (int i = 0; i < list1.size(); i++) {
    list3.add(list1.get(i) + list2.get(i));
}

System.out.println(list3);

File: data/conala-train_code-davinci-002_1666524963/Python/381.py
list1 = [1, 2, 3, 4, 5]
list2 = [6, 7, 8, 9, 10]

list3 = []

for i in range(len(list1)):
    list3.append(list1[i] + list2[i])

print(list3)